custom/icds_reports/templates/icds_reports/dashboard.html

```diff
@@ -5,22 +5,20 @@
 <!DOCTYPE html>
 <html>
 <head>
+  <link href="//netdna.bootstrapcdn.com/twitter-bootstrap/2.3.1/css/bootstrap-combined.min.css"/>
```

Comment:
it just occurred to me that this change in ordering could lead to unexpected style issues, though I'm somewhat inclined to leave it since presumably our styles should override the defaults.

i'm also confused why we are importing both bootstrap 2.3 and 3.3 which seems like a terrible idea. though am guessing nobody who works on the dashboard anymore could say.

Reply:
Hmm, yeah, that could happen. HQ's stylesheets don't override the basic bootstrap styles, they're more like tweaks to those basic styles. So rules that use exactly the same selector and appear in both 2 and 3 currently use the 2 styling, but after this PR will use the 3 styling. Unfortunately, I don't know a) how much overlap there is in exact rules between bootstrap 2 and 3 or b) of those overlapping rules, how many the dashboard uses.

Two suggestions to deal with this: either leave the ordering of stylesheets as it was and do a separate compress block for the sheets that appear before the 2.3.1 `bootstrap-combined.min.css`, or add bootstrap 2 to bower and then change the bootstrapcdn link to be local.

Agreed that importing both 2 and 3 is a bad idea and also that it should be dealt with separately.

Reply:
oh my god. when I added the `rel` and `type` attributes to the bootstrap 2.2 import the styles on the page completely broke. seems the browser was previously just ignoring it. :man_facepalming: 

I just deleted it.

Reply:
Let's call that a success...one less piece of tech debt to clean up.